packages/perseus-editor/src/util/register-all-widgets-and-editors-for-testing.test.ts

```diff
@@ -6,7 +6,9 @@ describe("Registering all widgets and editors", () => {
     it("should have an editor for every widget", () => {
         registerAllWidgetsAndEditorsForTesting();
 
-        const allWidgetsTypes = Widgets.getAllWidgetTypes();
+        const allWidgetsTypes = Widgets.getAllWidgetTypes()
+            // Auto correct is a special kind of widget, it DOES NOT need an editor
+            .filter((a) => a !== "deprecated-standin");
```

Comment:
I'm reviewing after the fact, but it would be nice if this widget used a consistent name everywhere.  The name is `deprecated-standin` but we refer to it as "auto correct".  Also, auto-correct is somewhat of a taken term to mean the hidden feature in our consuming apps where a developer can "speed run" an exercise by auto-answering the question. 

FWIW: I like `deprecated standin` as the name. 

I'll leave it with you to decide if it's worth changing.